runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -106,16 +107,7 @@ public void onConfigChanged(ConfigChanged event)
 			switch (event.getKey())
 			{
 				case CONFIG_KEY_FAIRY_RING_TOOLTIPS:
-					if (config.fairyRingTooltips())
-					{
-						Arrays.stream(FairyRingLocation.values())
-							.map(FairyRingLocation::getFairyRingPoint)
-							.forEach(worldMapPointManager::add);
-					}
-					else
-					{
-						worldMapPointManager.removeIf(FairyRingPoint.class::isInstance);
-					}
+					FairyRingLocation.setTooltip(config.fairyRingTooltips());
```

Comment:
What if you startup with both fairyRingTooltipls and fairyRingIcon disabled? The fairy ring locations will never get added